hypervisor/arch/x86/Kconfig

```diff
@@ -55,6 +55,12 @@ config SCHED_IORR
           IORR (IO sensitive Round Robin) scheduler supports multipule vCPUs running on
 	  on one pCPU, and they will be scheduled by a IO sensitive round robin policy.
 
+config SCHED_BVT
+        bool "BVT scheduler"
+        help
+          BVT (Borrowed Virtual Time) scheduler supports multipule vCPUs running on
+	  on one pCPU, and they will be scheduled based on virtual time.
```

Comment:
why on on?

Reply:
Add comments this is for better real time behavior?

Reply:
Add explicit statement to say BVT scheduler is built on top of prioritized scheduling mechanism, i.e. higher priority threads get scheduled first, and same priority tasks are scheduled per BVT.

Reply:
Will add more detail explanation, thanks.

Reply:
BVT is a general purpose scheduler, we may not say it is better for real time. It supports priority-based scheduling, so can work for real-time thread. Besides, the priority is achieved by 'warp' back virtual time. That means, a thread with larger warp value will have earlier virtual time, so can be dispatched firstly.

So, can I change it to :
BVT (Borrowed Virtual Time) is a virtual time based scheduling algorithm,
dispatching the runnable thread with the earliest effective virtual time. With
"warp" back in virtual time, it can provide prioritized scheduling mechanism,
and support real-time scenario.

Reply:
I thought you mis-understood. In general, ACRN uses prioritized scheduling algorithm. The thread with higher priority got scheduled earlier than lower priority thread. This is an hard requirement for RT usage.

However, if 2 threads have same priority, we use BVT to schedule between them.